src/content/advanced/automatic-node-termination/index.md

```diff
@@ -30,11 +32,11 @@ An unhealthy status means the `kubelet` on a given node has reported `NotReady`
 
 ## Enabling automatic termination
 
-Automatic termination of unhealthy nodes is **disabled by default**. You can enable it for each individual cluster.
+Automatic termination of unhealthy nodes is **enabled by default** for releases greater or equal to `v15.0.0` nd  **disabled by default** for older releases.
```

Comment:
```suggestion
Automatic termination of unhealthy nodes is **enabled by default** on AWS since release `v15.0.0` and  **disabled by default** for older releases.
```